### ISSUE TITLE
hcaptcha - rqdata + useragent

### ISSUE THREAD

**Issue by NotGoodCoding** [dzmitry-duboyski/2captcha-ts#1]
Hi, did you implement rqdata + useragent parameters? If so, how would I construct the request on include this?

Thanks.

**Comment by dzmitry-duboyski** [comment]
@NotGoodCoding hi. 
Yes, you can send `rqdata` using the `data` parameter.
Note that when sending `rqdata`, you must also send the `invisible` and `userAgent` parameters.

Here is an example of how to send hCaptcha in your case:
```JavaScript
const solver = new Captcha.Solver(APIKEY);

solver.hcaptcha({
    pageurl: "https://2captcha.com/demo/hcaptcha?difficulty=moderate",
    sitekey: "b76cd927-d266-4cfb-a328-3b03ae07ded6",
    invisible: 1,
    data: "the_rqdata_value_you_found",
    userAgent: "User-Agent_that_you_use_when_interacting_with_the_site"
})
.then((res) => {
    console.log(res);
})
.catch((err) => {
    console.log(err);
})
```
Please check if this solution worked for you and let me know if there are any problems.

**Comment by dzmitry-duboyski** [comment]
@NotGoodCoding Hello again. Today I released [v1.3.1](https://github.com/dzmitry-duboyski/2captcha-ts/releases/tag/v1.3.1) in which I updated the JSDOC documentation for methods. Added the missing parameters for methods, including information about the `rqdata` and `useragent` parameters for method `hcaptcha`.

Screenshot:
![chaptcha](https://user-images.githubusercontent.com/38065632/222960306-0f42ec55-dc06-4cb9-85d6-a7074691dff6.gif)

**Comment by NotGoodCoding** [comment]
> @NotGoodCoding Hello again. Today I released [v1.3.1](https://github.com/dzmitry-duboyski/2captcha-ts/releases/tag/v1.3.1) in which I updated the JSDOC documentation for methods. Added the missing parameters for methods, including information about the `rqdata` and `useragent` parameters for method `hcaptcha`.
> 
> Screenshot: ![chaptcha](https://user-images.githubusercontent.com/38065632/222960306-0f42ec55-dc06-4cb9-85d6-a7074691dff6.gif) [ ![chaptcha](https://user-images.githubusercontent.com/38065632/222960306-0f42ec55-dc06-4cb9-85d6-a7074691dff6.gif) ](https://user-images.githubusercontent.com/38065632/222960306-0f42ec55-dc06-4cb9-85d6-a7074691dff6.gif) [ ](https://user-images.githubusercontent.com/38065632/222960306-0f42ec55-dc06-4cb9-85d6-a7074691dff6.gif)

Thanks a lot!!!

**Comment by dzmitry-duboyski** [comment]
@NotGoodCoding you're welcome.
